src/templates/TemplateData.ts

```diff
@@ -147,29 +168,37 @@ export async function tryGetTemplateDataFromCache(reporter: TelemetryReporter |
     }
 }
 
-export async function tryGetTemplateDataFromFuncPortal(reporter: TelemetryReporter | undefined, globalState?: vscode.Memento, hostname: string = 'functions.azure.com'): Promise<TemplateData | undefined> {
+export async function tryGetTemplateDataFromFuncPortal(reporter: TelemetryReporter | undefined, globalState?: vscode.Memento): Promise<TemplateData | undefined> {
```

Comment:
How about we rename this to 'tryGetLatestTemplateData'

I'm not sure 'FuncPortal' still has the same meaning